evalml/tests/component_tests/test_featuretools.py

```diff
@@ -163,3 +164,55 @@ def test_dfs_sets_max_depth_1(mock_dfs, X_y_multi):
     feature.fit(X_pd, y)
     _, kwargs = mock_dfs.call_args
     assert kwargs["max_depth"] == 1
+
+
+@patch("evalml.pipelines.components.transformers.preprocessing.featuretools.dfs")
+def test_dfs_with_serialized_features(mock_dfs, X_y_binary):
```

Comment:
this test checks that DFS is not called when features are serialized and that the feature matrix is computed correctly using serialized features.